create_xcframeworks.sh

```diff
@@ -0,0 +1,47 @@
+mkdir output
+mkdir output/archives
+mkdir output/iphoneos
+mkdir output/macosx
```

Comment:
It would be nice to add a common header to the script:

```bash
#!/usr/bin/env bash
#
# Shebang to explicitly use bash and not break on macOS boxes with zsh as default shell.
#
# After that you usually describe what this script is for and how to use it. In particular,
# what it expects the current directory to be. (That's more important if it's it "scripts".)

set -eu # common flags to ensure that the shell does not ignore failures
```

I'd also put all temporary things into the `build` directory (overridable by `BUILD_PATH` environment variable). Native build system places its stuff there, that directory is already in gitignore, etc.